README.md

```diff
@@ -114,6 +165,6 @@ Your Answer:
 Select all that apply:
 ```
 [] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing.
-[] Synchronous code runs all at the same time, asynchronous code runs completely randomly
-[] Synchronous code runs in order (as appears in the source), asynchronous code may run at a later time.
+[x] Synchronous code runs all at the same time, asynchronous code runs completely randomly
```

Comment:
It would actually only be the last answer here. Asynchronous code runs based on timing, but it's not random. Synchronous code runs in order as it appears in the script file